### PR TITLE
Tomcat 8 template tweaks

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: templates/default/instances/conf/server.xml.erb [hunk 1]
```diff
@@ -25,13 +25,12 @@
...
  -->
 <Server port="-1" shutdown="SHUTDOWN">
+  <Listener className="org.apache.catalina.startup.VersionLoggerListener" />
   <!-- Security listener. Documentation at /docs/config/listeners.html
   <Listener className="org.apache.catalina.security.SecurityListener" />
   -->
   <!--APR library loader. Documentation at /docs/apr.html -->
   <Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on" />
-  <!--Initialize Jasper prior to webapps are loaded. Documentation at /docs/jasper-howto.html -->
-  <Listener className="org.apache.catalina.core.JasperListener" />
```
Comment (isuftin) [inline templates/default/instances/conf/server.xml.erb]: What was the purpose of removing this?

File: templates/default/instances/conf/server.xml.erb [hunk 2]
```diff
@@ -25,13 +25,12 @@
      Documentation at /docs/config/server.html
  -->
 <Server port="-1" shutdown="SHUTDOWN">
+  <Listener className="org.apache.catalina.startup.VersionLoggerListener" />
```
Comment (isuftin) [inline templates/default/instances/conf/server.xml.erb]: Will this work for Tomcat 7?

File: templates/default/instances/conf/server.xml.erb (hunk 1 above)
Comment (thongsav-usgs) [inline templates/default/instances/conf/server.xml.erb]: Tomcat 8 doesn't use it.

File: templates/default/instances/conf/server.xml.erb (hunk 2 above)
Comment (thongsav-usgs) [inline templates/default/instances/conf/server.xml.erb]: No, I was contemplating if I wanted to make 0.1.3 work with 7, but just went with making it Tomcat 8 only. Would be simple enough to add a toggle based on major version number.

File: templates/default/instances/conf/server.xml.erb (hunk 1 above)
Comment (isuftin) [inline templates/default/instances/conf/server.xml.erb]: But Tomcat 7 does. This should probably be included if the installation is Tomcat 7 and removed if the installation is Tomcat 8

File: templates/default/instances/conf/server.xml.erb (hunk 2 above)
Comment (isuftin) [inline templates/default/instances/conf/server.xml.erb]: Let's do a toggle.

**Pull request merged**